### DISCUSSION TITLE
[Feature] Share photos from other apps to Immich

### DISCUSSION THREAD

**Discussion by eygraber** [immich-app/immich#5172] [Feature Request]
### The feature

Occasionally I'll receive a photo on WhatsApp or some other app that I'd like to share to Immich so that it gets uploaded to my server. I don't want to use the backup feature for this because it is ad-hoc. Is it currently possible to do that, and if not, is it already on a roadmap?

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2659
